wgsl/index.bs

```diff
@@ -6059,6 +6059,10 @@ control to an enclosing [=statement/continuing=] statement.
 A `continue` statement must not be placed such that it would transfer
 control past a declaration used in the targeted [=statement/continuing=] statement.
 
+Note: A `continue` can only be used in a `continuing` statement if it is used for transferring control
```

Comment:
This restates rule from the sentence on line 6055.

I guess a second way, as a note is not harmful, and could help.